standalone-metastore/metastore-tools/metastore-benchmarks/pom.xml

```diff
@@ -88,6 +88,12 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
The slf4j-log4j12 is an explicit binding of slf4j with log4j 1.2 thus this exclusion here doesn't make sense. The whole dependency must be dropped.

Just below we have log4j-slf4j-impl which is an SLF4J binding with Log4j 2 so I think that's all we need.

Reply:
understood.